mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/profile/ProfileState.kt

```diff
@@ -9,4 +10,19 @@ interface ProfileState : Person {
 
   /** The email address of the currently connected user. */
   val email: String
+
+  /** Number of past games */
+  val numberOfGames: Int
+
+  /** Number of puzzles */
+  val numberOfPuzzles: Int
```

Comment:
I don't think it's very common to have properties named `...Of...` in Kotlin. I'd suggest using a name like `gamesCount` or `pastGamesCount`.

Reply:
👍